engines/bastion/README.md

```diff
@@ -188,13 +188,13 @@ Open the file, and add the following lines (with empty lines above and below for
 To setup a development environment, clone the repository or your fork of the repository. From the git checkout, setup the required development dependencies by running:
 
 ```
-rake bastion:setup
+npm install
 ```
 
 After making changes, tests and linting can be run via:
 
 ```
-rake bastion:ci
+grunt ci
```

Comment:
Do you plan on adding these to Katello's `package.json`?

Reply:
after @ehelm's suggestion, i decided to not do that (sorry should have mentioned that here).  I'd rather get this in quickly and then we can adjust if desired.  Does that sound okay?

Reply:
no worries, gotta draw the line somewhere :) I guess there is a difference between testing bastion and bastion_katello and I'm forgetting this too! 

I'll try to do a follow up PR adding these to `package.json`, maybe we could have a single `npm run ci` command to run everything we expect jenkins to do as well.

Reply:
Just be cautious given they both have their own package.json with dependencies and pathing configured based on their relative locations. Mixing these with the base katello dependencies and scripts might be more more mashing together than you necessarily want. If anything, I'd think you'd be better off investing in dropping all of the Grunt infrastructure in favor of native scripts (but even that may not be worth the time if the webpack/React is the future).